back/engines/commercial/user_custom_fields/spec/acceptance/stats_users_spec.rb

```diff
@@ -66,8 +66,8 @@ def xlsx_worksheet_to_array(worksheet)
     worksheet.map { |row| row.cells.map(&:value) }
   end
 
-  describe 'by_gender endpoints' do
-    get 'web_api/v1/stats/users_by_gender' do
+  describe 'former by_gender endpoints' do
```

Comment:
I would just remove those tests if they tests exactly the same thing as the select case for `get 'web_api/v1/stats/users_by_custom_field/:custom_field_id'`. Otherwise, I would move them to the `web_api/v1/stats/users_by_custom_field/:custom_field_id` in a new `describe` block, something like `with gender custom field`.